src/main/java/com/google/songgame/servlets/GameServlet.java

```diff
@@ -35,6 +38,12 @@
   private static final int ROUND_LENGTH = 30000;
   private static final long MAX_RESULTS = 25L;
   private String videoId;
```

Comment:
Is this variable necessary anymore after your changes? If not, can you delete it?

Reply:
still think this comment needs to be addressed